docs/source/user_guide/installation/install-source-windows.rst

```diff
@@ -0,0 +1,382 @@
+	:scale: 100%
+	:alt: Image with the packages to install
+
+
+VCPKG
+~~~~~
+
+PyBaMM comes with a DAE solver based on the IDA solver provided by the SUNDIALS library.
+To use this solver, you must ensure you have the necessary SUNDIALS components
+installed on your system.
+
+VCPKG automatically installs them for you during the build process. To learn more, look at the `official documentation by Microsoft <https://learn.microsoft.com/en-us/vcpkg/get_started/overview/>`_.
+
+To install VCPKG, follow the steps below:
+
+1. Open a Command Shell, and navigate to ``C:\``.
+    * We are installing VCPKG in ``C:\vcpkg``
+    * If you wish to install in a custom directory, change the environment variable ``VCPKG_ROOT_DIR`` before installation as mentioned herein :ref:`installing-pybamm`.
+2. Clone VCPKG's GitHub repository
+
+   .. code:: bash
+
+	  git clone https://github.com/microsoft/vcpkg.git
+
+
+   * If you do not have Git installed, download and install it from the `official Git download page <https://git-scm.com/download/win>`_.
+
+3. Navigate inside VCPKG directory and install it:
+
+   .. code:: bash
+
+	  cd vcpkg && bootstrap-vcpkg.bat
+
+4. Add VCPKG directory to ``Path`` environment variable:
+    * Copy the location ``C:\vcpkg`` or the location of your custom directory.
+    * In the start menu, search for ``Edit the system environment variables`` and open it.
+    * Under the ``Advanced`` tab, click on ``Environment Variables``
+        .. image:: images-windows-source/pic2-env-var-open.png
+			:scale: 70%
+
+
+    * Select ``Path`` from ``user variables`` if you want to set VCPKG just for you, or use ``Path`` from ``system variables`` to set VCPKG for all your system users.
+    	.. image:: images-windows-source/pic3-show-path.png
+		    :scale: 50%
+
+
+    * Select ``Path`` and select ``NEW``. Paste the location to VCPKG, then confirm your options by clicking ``OK``.
+    	.. image:: images-windows-source/pic4-save-env-var.png
+		  :height: 900px
+		  :width: 900px
+		  :scale: 70%
+
+
+5. Close and reopen Command Prompt and execute ``vcpkg`` to see an output of different options available for ``vcpkg``.
+
+CMake
+~~~~~
+
+``CMake`` is required to install the SUNDIALS and other libraries for the ``IDAKLU`` solver.
+To install it, follow the link to the `official CMake downloads page <https://cmake.org/download/>`_.
+
+Download an installer based on your system's architecture, i.e. ``x32/x64``, and check ``Add CMake to the PATH environment variable`` during installation.
+
+Python Packages
+~~~~~~~~~~~~~~~
+
+In addition to the packages above, you will need `TOML Kit <https://tomlkit.readthedocs.io/en/latest/>`_, a ``toml`` table parser. The tool is necessary to install build-time dependencies in "editable mode" without build isolation.
+
+To install ``tomlkit`` to your local user account (ensure you are not within a virtual environment), use the following command:
+
+.. code:: bash
+
+	  python3.X -m pip install --user tomlkit
```

Comment:
It has been a while since I used windows, but I don't recall having to be so specific to the exact python version. I remember just `python` or `python3` working. Python 2 is not even supported anymore
```
	  python -m pip install --user tomlkit
```
or
```
	  python3 -m pip install --user tomlkit
```

Reply:
You are right. I was under the presumption that the users had other lower Python versions. We can  edit this.